oci-seccomp-bpf-hook.go

```diff
@@ -146,9 +151,8 @@ func main() {
 	log.SetFormatter(formatter)
 	log.SetOutput(logfile)
 	if *runBPF > 0 {
-		log.Println("Filepath : ", profilePath)
-		if err := runBPFSource(*runBPF, profilePath, log); err != nil {
-			log.Error(err)
+		if err := runBPFSource(*runBPF, outputProfilePath, *inputFile, log); err != nil {
+			logrus.Error(err)
```

Comment:
Why not `return err` here?

Reply:
This should return error.

Reply:
I cannot `return err` as the function is called in the `main()`, `runBPFSource` returns an error, I should make it `log.Fatal(err)` instead of `log.Error(err)`